### PR TITLE
Add a simple backend server wrapping the CLI tool

### PR DESCRIPTION
**Pull request by fatadel**

### CODE REVIEW

File: opeanapi-spec.yaml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                    code:
+                      ts: "http://127.0.0.1:3000/54a3f07fc9135e44/code/54a3f07fc9135e44.ts"
+                      tsconfig: "http://127.0.0.1:3000/54a3f07fc9135e44/tsconfig.json"
+                      js: "http://127.0.0.1:3000/54a3f07fc9135e44/code/54a3f07fc9135e44_index.js"
+                    sd: "http://127.0.0.1:3000/54a3f07fc9135e44/sds/54a3f07fc9135e44.json"
+                System Description To Sequence Diagram:
+                  value:
+                    success: true
+                    code:
+                      ts: "http://127.0.0.1:3000/d7db70f9a3cb4bd1/code/d7db70f9a3cb4bd1.ts"
```
Comment (egekorkan) [inline opeanapi-spec.yaml:98]: shouldn't this be a variable?

**Review by egekorkan — CHANGES_REQUESTED** [review]
My actual feedback would be to not require .puml files to be sent as URL encoded strings. People generally use an editor to write PlantUML files and ideally they should be able to just copy paste as raw data in postman in a POST request. What is the reason for choosing URL Encoding?

File: opeanapi-spec.yaml (hunk 1 above)
Comment (fatadel) [reply opeanapi-spec.yaml:98]: Sorry, what kind of variable do you mean?

**Comment by fatadel** [comment]
> My actual feedback would be to not require .puml files to be sent as URL encoded strings. People generally use an editor to write PlantUML files and ideally they should be able to just copy paste as raw data in postman in a POST request. What is the reason for choosing URL Encoding?

Better now? 😊

**Review by egekorkan — APPROVED** [review]

**Comment by egekorkan** [comment]
I think that importing the ts files like the way cli does would have been nicer but this is indeed a wrapper and can be considered an initial step so I will merge it :)

**Pull request merged**